app/main.py

```diff
@@ -16,4 +16,21 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+
+def sort_variables() -> dict:
```

Comment:
You just need to write a dict here with two keys - mutable and immutable

Reply:
Did it